mongodb-queue.ts

```diff
@@ -130,7 +130,7 @@ export class MongoDBQueue<T = any> {
       visible: {$lte: now()},
     };
     const sort: Sort = {
-      _id: 1,
```

Comment:
Definitely a different change, which will need an index regeneration.

I'm also not sure it's needed, since I think in theory MongoDB can still leverage the index for a [non-prefix sort](https://www.mongodb.com/docs/manual/tutorial/sort-results-with-indexes/#sort-and-non-prefix-subset-of-an-index) since `deleted: {$exists: false}` [counts as an equality condition](https://stackoverflow.com/a/31372552/4003671).